src/components/tools/polygon/PolygonTool.vue

```diff
@@ -109,10 +127,17 @@ export default defineComponent({
       placingTool.remove();
     });
 
+    const keys = useMagicKeys();
+    const mergeKey = computed(() => keys[actionToKey.value.mergeNewPolygon]);
```

Comment:
I think this should be `computed(() => keys[actionToKey.value.mergeNewPolygon].value)`.

Reply:
Yes.